vision/lidar/view_recording.py

```diff
@@ -0,0 +1,220 @@
+            range_img = np.clip(range_img * 1.8, 0, 255).astype(np.uint8)  # increase brightness
+
+            cv2.imshow(
+                f"Range, (Upscale: {hscale}x width, {vscale}x height)",
+                cv2.resize(  # resize image to fit screen
+                    range_img,
+                    (range_img.shape[1] * hscale, range_img.shape[0] * vscale),
+                    interpolation=cv2.INTER_AREA,
+                ),
+            )
+
+        if viz_flags.get("reflectivity"):
+            # Reflectivity visualization
+            refl_field: NDArray[Shape["32, 1024"], UInt32]
+            refl_field = scan.field(client.ChanField.REFLECTIVITY)
+            refl: NDArray[Shape["32, 1024"], UInt8] = client.destagger(metadata, refl_field)
+            refl = (refl / np.max(refl) * 255).astype(np.uint8)  # normalize image
+            refl = np.clip(refl * 2.2, 0, 255).astype(np.uint8)  # increase brightness
+            cv2.imshow(
+                f"Reflectivity, (Upscale: {hscale}x width, {vscale}x height)",
+                cv2.resize(
+                    refl,
+                    (refl.shape[1] * hscale, refl.shape[0] * vscale),
+                    interpolation=cv2.INTER_AREA,
+                ),
+            )
+
+        if viz_flags.get("signal"):
+            # Signal visualization
+            signal_field: NDArray[Shape["32, 1024"], UInt32] = scan.field(client.ChanField.SIGNAL)
+            signal: NDArray[Shape["32, 1024"], UInt8] = client.destagger(metadata, signal_field)
+            signal = (signal / np.max(signal) * 255).astype(np.uint8)  # normalize image
+            signal = np.clip(signal * 2.2, 0, 255).astype(np.uint8)  # increase brightness
+            cv2.imshow(
+                f"Signal, (Upscale: {hscale}x width, {vscale}x height)",
+                cv2.resize(
+                    signal,
+                    (signal.shape[1] * hscale, signal.shape[0] * vscale),
+                    interpolation=cv2.INTER_AREA,
+                ),
+            )
+
+        if viz_flags.get("nearir"):
+            # Near infrared visualization
+            nearir_field: NDArray[Shape["32, 1024"], UInt32] = scan.field(client.ChanField.NEAR_IR)
+            nearir: NDArray[Shape["32, 1024"], UInt8] = client.destagger(metadata, nearir_field)
+            nearir = (nearir / np.max(nearir) * 255).astype(np.uint8)  # normalize image
+            nearir = np.clip(nearir * 1.5, 0, 255).astype(np.uint8)  # increase brightness
+            cv2.imshow(
+                f"Near-IR, (Upscale: {hscale}x width, {vscale}x height)",
+                cv2.resize(
+                    nearir,
+                    (nearir.shape[1] * hscale, nearir.shape[0] * vscale),
+                    interpolation=cv2.INTER_AREA,
+                ),
+            )
+
+        cv2.waitKey(25)  # modify this delay (in ms) if the visualization is too fast or too slow
```

Comment:
Should this be a constant at the top of the function to make it clearer/more accessible?